app/views/hub/sites/_form.html.erb

```diff
@@ -7,6 +6,21 @@
     <%= f.cfa_input_field :name, t("general.name") %>
     <%= f.cfa_select :parent_organization_id, t("general.organization"), @organizations.map { |c| [c.name, c.id] } %>
 
+    <% if @show_unique_links %>
+      <% @site.source_parameters.each do |source_parameter| %>
+        <%= f.fields_for :source_parameters, source_parameter, builder: VitaMinFormBuilder do |source_parameter_form| %>
+          <%= source_parameter_form.cfa_input_field :code, t("general.unique_link"), classes: ["form-width--short"] %>
+          <%= source_parameter_form.cfa_checkbox(:_destroy, t("general.remove")) %>
+        <% end %>
+      <% end %>
+
+      <h3><%= f.label t("general.new") %></h3>
```

Comment:
This sections feel like it should be a "New" button, that clicking that opens a new text field to add N more Unique Urls. If this is only used internally it's probably fine however.

Reply:
Agree on both points.